.travis.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mongo --quiet -p 27017 --eval 'db.version();'
```

Comment:
what's the purpose of running `db.version()` here?

Reply:
It helps with debugging - in case Travis decides to update and the tests start failing for some unexplained reason.